src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesArray = sourceString
```

Comment:
[CODE KNOWLEDGE] - There is no need to make a copy of the array before the `map` operation, as `map` does not mutate the original array. You can start chaining directly with the `split` method.